CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,24 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    public var titleFont: UIFont = UIFont.systemFontOfSize(16) {
+        didSet {
+            updateTabs()
+        }
+    }
+    
     @IBInspectable public var lightBackground: Bool = true {
         didSet {
             updateBackground()
         }
     }
     
+    @IBInspectable public var darkBackground: Bool = false {
```

Comment:
Why add a second variable for this? Dark is the inverse of light, so `false` on `lightBackground` means dark is `true`.